cmd/autoowners/main.go

```diff
@@ -401,6 +440,11 @@ func getTitle(matchTitle, datetime string) string {
 }
 
 func main() {
+	if os.Getenv("autoowners_debug_mode") == "true" {
```

Comment:
If we want to commit this, let's have it as a flag...

Reply:
Yes. Done.